amcl/src/amcl_node.cpp

```diff
@@ -919,35 +1055,49 @@ AmclNode::convertMap( const nav_msgs::OccupancyGrid& map_msg )
       map->cells[i].occ_state = +1;
     else
       map->cells[i].occ_state = 0;
+  //ROS_DEBUG("%i\n", map->cells[i].occ_state);
   }
 
   return map;
 }
 
 AmclNode::~AmclNode()
 {
-  delete dsrv_;
+  //delete dsrv_;
```

Comment:
This and the delete's below look like they should be commented back in? I don't see an alternative cleanup delete.